arclight-neoforge/src/main/java/io/izzel/arclight/neoforge/mixin/core/world/item/ItemStackMixin_NeoForge.java

```diff
@@ -64,7 +69,7 @@ public abstract class ItemStackMixin_NeoForge implements ItemStackBridge, IItemS
     }
 
     @Deprecated
-    public void setItem(Item item) {
+    public void setItem(@Nullable Item item) {
```

Comment:
As far as I know, there shouldn't be an `ItemStack` with a null item, it should be `Items.AIR` instead.